content/news/XXXX-XX-XX-mixxx-2.4.2-released.md

```diff
@@ -0,0 +1,37 @@
+title: Mixxx 2.4.2 Release Announcement
+authors: Evelynne Veys
+tags: update, 2.4.2
+comments: yes
+status: draft
+
+#### Winter and Summer are coming
+
+Contrasts.
+As the leaves fall and the first snowflakes start drifting down in the North, while the southern skies start warming up
+with summer sun, we are thrilled to finally release Mixxx 2.4.2! Finally because a bit later than planned, we would like
+to apologize for the delay but we can't release a non-prefect version. This release is a major bugfix-release for the

```

Comment:
no need to apologize, let's be proud instead.

Reply:
fixed

Reply:
This paragraph is now partially duplicated.

Reply:
still busy, deleted all website on my harddrive to see if that would solve , made another pr still no bulletlist....